pkg/uefi/Dockerfile

```diff
@@ -15,16 +15,31 @@ ENV BUILD_PKGS_amd64 iasl
 ENV BUILD_PKGS_arm64 iasl
 RUN eve-alpine-deploy.sh
 
-ENV EDK_COMMIT ca407c7246bf405da6d9b1b9d93e5e7f17b4b1f9
+ENV EDK_VERSION_aarch64 edk2-stable202208
+ENV EDK_COMMIT_aarch64 edk2-stable202208
+
+ENV EDK_VERSION_x86_64 edk2-stable202005
+ENV EDK_COMMIT_x86_64 ca407c7246bf405da6d9b1b9d93e5e7f17b4b1f9
+
 ENV SBI_COMMIT cbaa9b0333517b3c25bea8d1c71ac8005ff1f727
 
-RUN git clone -b edk2-stable202005 https://github.com/tianocore/edk2.git /ws && git -C /ws checkout ${EDK_COMMIT}
-RUN git clone https://github.com/riscv/opensbi.git /opensbi && git -C /opensbi checkout ${SBI_COMMIT}
+RUN if [ "$(uname -m)" = aarch64 ] || [ "$(uname -m)" = x86_64 ]; then \
+        git clone -b "$(eval echo \$EDK_VERSION_"$(uname -m)")" https://github.com/tianocore/edk2.git /ws && \
+        git -C /ws checkout "$(eval echo \$EDK_COMMIT_"$(uname -m)")"; \
+    fi
+RUN if [ "$(uname -m)" = riscv64 ]; then \
+        git clone https://github.com/riscv/opensbi.git /opensbi && \
```

Comment:
@giggsoff for the riscv64, we use opensbi, not the edk2. In build.sh we are still build everything we need for riscv64.

Reply:
Ah, you are right, sorry.
And sorry again, seems we have no automation yet to check builds on riscv64 in CI before merge. But can you please double-check it with `make ZARCH=riscv64 pkg/uefi` as you changed the arch-specific logic? I can see the problem on `RUN git submodule update --init` as we are not in git directory on that step with your changes.

Reply:
Yes, you are right, I completely forgot about updating submodules, thanks!